mne/viz/_brain/_brain.py

```diff
@@ -2342,6 +2346,42 @@ def add_head(self, dense=True, color=None, alpha=0.5):
         surf = _get_head_surface('seghead' if dense else 'head',
                                  self._subject_id, self._subjects_dir)
         verts, triangles = surf['rr'], surf['tris']
+        verts *= 1e3 if self._units == 'mm' else 1
+        if color is None:
+            color = DEFAULTS['coreg']['head_color']
```

Comment:
This was added in this cycle so I would not have this, I'd remove support for `None` entirely